lib/protoparser/vmimport/parser.go

```diff
@@ -103,6 +104,31 @@ func (r *Row) unmarshal(s string, tu *tagsUnmarshaler) error {
 	return nil
 }
 
+func getFloat64(value *fastjson.Value) (float64, error) {
+	if value == nil {
+		return 0, fmt.Errorf("value is empty")
+	}
+
+	if value.Type() == fastjson.TypeString {
+		strVal := strings.ReplaceAll(strings.ToLower(value.String()), "\"", "")
+		if strings.EqualFold(strVal, "infinity") {
```

Comment:
doesn't `EqualFold` compare strings irrespectively to their case? If yes, why would you need L:113. 
Or wise versa, if you already manually convert string to lower case - why would you need `EqualFold`?
Have you tested what would be more efficient:
1. Check each value multiple times via EqualFold 
2. Or lowercase value once and then do a switch-case?